docs/api.md

```diff
@@ -1440,6 +1441,77 @@ Failing requests may return the following error:
 
 * status code 400, errno 102: unknown account
 
+## POST /v1/account/devices/notify
+
+:lock: HAWK-authenticated with the sessionToken.
+
+Notifies a set of devices in the caller's account of an event
+by sending a Push notification.
+
+### Request
+
+___Headers___
+
+The request must include a Hawk header that authenticates the request
+using a `sessionToken` received from `/v1/account/create` or `/v1/account/login`.
+
+#### Notify all other devices except excluded devices identified by their id
+
+```sh
+curl -v \
+-X POST \
+-H "Host: api-accounts.dev.lcip.org" \
+-H "Content-Type: application/json" \
+-H 'Authorization: Hawk id="d4c5b1e3f5791ef83896c27519979b93a45e6d0da34c7509c5632ac35b28b48d", ts="1373391043", nonce="ohQjqb", hash="vBODPWhDhiRWM4tmI9qp+np+3aoqEFzdGuGk0h7bh9w=", mac="LAnpP3P2PXelC6hUoUaHP72nCqY5Iibaa3eeiGBqIIU="' \
+https://api-accounts.dev.lcip.org/v1/account/devices/notify \
+-d '{
+  "to": "all",
+  "excluded": ["0f7aa00356e5416e82b3bef7bc409eef"],
+  "payload": {
+    version: 1,
+    command: "sync:collection_changed",
+    data: {
+      collections: ["clients"]
+    }
+  },
+  "TTL": 10 // In seconds, defaults to 0
+}'
+```
+
+#### Notify specific devices identified by their id
+
+```sh
+curl -v \
+-X POST \
+-H "Host: api-accounts.dev.lcip.org" \
+-H "Content-Type: application/json" \
+-H 'Authorization: Hawk id="d4c5b1e3f5791ef83896c27519979b93a45e6d0da34c7509c5632ac35b28b48d", ts="1373391043", nonce="ohQjqb", hash="vBODPWhDhiRWM4tmI9qp+np+3aoqEFzdGuGk0h7bh9w=", mac="LAnpP3P2PXelC6hUoUaHP72nCqY5Iibaa3eeiGBqIIU="' \
+https://api-accounts.dev.lcip.org/v1/account/devices/notify \
+-d '{
+  "to": ["0f7aa00356e5416e82b3bef7bc409eef", "fee904cb7feb3b28e6145e65300aa7f0"],
+  "payload": {
+    version: 1,
+    command: "sync:collection_changed",
+    data: {
+      collections: ["clients"]
+    }
+  }
+}'
+```
+
+### Response
+
+Successful requests will return a `200 OK` response
+with an empty object in the JSON body:
+
+```json
+{}
+```
+
+Failing requests may return the following error:
+
+* status code 400, errno 107: may be sent if the payload parameter is not valid (see [pushpayloads.schema.json](pushpayloads.schema.json))
```

Comment:
We will if we want to enforce some sort of server-side rate-limiting (which we should probably discuss more broadly, and can be in a follow-up PR if you like)